sympy/functions/special/error_functions.py

```diff
@@ -1895,6 +1895,29 @@ def _eval_rewrite_as_sinc(self, z, **kwargs):
         t = Symbol('t', Dummy=True)
         return Integral(sinc(t), (t, 0, z))
 
+    def _eval_as_leading_term(self, x, logx=None, cdir=0):
+        arg = self.args[0].as_leading_term(x)
+        arg0 = arg.subs(x, 0)
+
+        if arg0 is S.NaN:
+            arg0 = arg.limit(x, 0, dir='-' if re(cdir).is_negative else '+')
+        if arg0.is_zero:
+            return arg
+        elif not arg0.is_infinite:
+            return self.func(arg0)
+        else:
+            return self
+    
+
+    def test_si():
```

Comment:
Wait I'm confused.  The leading term implementation looks correct to me but then the relevant tests should be added to the test file (`test_error_functions.py` I reckon) . Why have the tests been added here ?